virtool/jobs/db.py

```diff
@@ -61,3 +63,14 @@ async def processor(db, document: dict) -> dict:
         ),
         [AttachUserTransform(db)],
     )
+
+
+async def fetch_complete_job(db, document, key=None) -> Job:
+    document = await processor(db, document)
+    document["user"] = UserNested(**document["user"])
+    document["status"] = [JobStatus(**status) for status in document["status"]]
```

Comment:
Is this necessary? I would think that Pydantic would deal with making the objects for you.

Reply:
You are right, I think I was missing parameters earlier which caused it not to work.